src/combinatorics/simple_graph/subgraph.lean

```diff
@@ -262,9 +262,30 @@ instance : bounded_order (subgraph G) :=
 
 @[simp] lemma not_bot_adj {v w : V} : ¬(⊥ : subgraph G).adj v w := not_false
 
+@[simp] lemma inf_adj {H₁ H₂ : subgraph G} {v w : V} :
+  (H₁ ⊓ H₂).adj v w ↔ H₁.adj v w ∧ H₂.adj v w := iff.rfl
+
+@[simp] lemma sup_adj {H₁ H₂ : subgraph G} {v w : V} :
+  (H₁ ⊔ H₂).adj v w ↔ H₁.adj v w ∨ H₂.adj v w := iff.rfl
+
+@[simp] lemma edge_set_top : (⊤ : subgraph G).edge_set = G.edge_set :=
+set.ext (λ e, sym2.ind (by simp) e)
+
+@[simp] lemma edge_set_bot : (⊥ : subgraph G).edge_set = ∅ :=
+set.ext (λ e, sym2.ind (by simp) e)
```

Comment:
This and the following can be replaced like so if you want.
```suggestion
set.ext $ sym2.ind (by simp)
```

Reply:
I thought I tried this, apparently I didn't! Thanks

Reply:
You probably did try it and it didn't work, and I tried something *very* similar the other day to no avail. Recursor elaboration is fickle.